### PR TITLE
chore(HOME-3495): v10 Redo Account Model Raml

### PR DESCRIPTION
**Pull request by kongmalin**
Closes: HOME-3495

### CODE REVIEW

**Comment by kongmalin** [comment]
screen shot 2018-05-17 at 9 38 17 pm
screen shot 2018-05-17 at 9 38 25 pm

File: api/v10/examples/accounts.res.json [hunk 1]
```diff
@@ -1,130 +1,38 @@
 {
-  "accounts": [
```
Comment (yuan616510) [inline api/v10/examples/accounts.res.json]: why changed a list to a single object?

**Comment by yuan616510** [comment]
Need to add 204 response for index.raml

**Comment by yuan616510** [comment]
One more thing, I checked v10 wiki page that said "route exists need to change to drop plural". Is that change in a separate PR? 

https://trizic.atlassian.net/wiki/spaces/DEV/pages/475955263/v10+routes

**Comment by kongmalin** [comment]
Yes, I put in HOME-3490

**Pull request merged**